generator.json

```diff
@@ -1328,7 +1329,8 @@
                     "d3d11_4.h": "[Core]D3D11",
                     "d3d11shader.h": "[Core]D3D11",
                     "d3d11shadertracing.h": "[Core]D3D11",
-                    "d3d11sdklayers.h": "[Core]D3D11"
+                    "d3d11sdklayers.h": "[Core]D3D11",
+                    "d3d11on12.h": "[Core]D3D11"
```

Comment:
You haven't staged generator.json

Reply:
a my bad

Reply:
to be more accurate i actually have a separate json file i paste a single job into so i dont wait multiple mins per iteration on the generation, i just forgot to copy back to generator.json